databuilder/models/table_metadata.py

```diff
@@ -17,6 +17,55 @@
 DESCRIPTION_NODE_LABEL = DESCRIPTION_NODE_LABEL_VAL
 
 
+class BadgeMetadata(Neo4jCsvSerializable):
```

Comment:
discuss offline, the next step is to improve this badge model to build:
1. badge<->column relationship
2. badge<->table relationship
3. badge<->dashboard relationship